source/docs/articles/local/git-faq.md

```diff
@@ -5,11 +5,17 @@ category:
   - developing
 keywords: git, git commands, conflicts, resolve conflicts, core, drupal, wordpress
 ---
+[Git](https://git-scm.com/) is the version control tool at the heart of the Pantheon workflow. If you're a developer who likes to use [local development](/docs/articles/local), it's a good way to work with the Pantheon platform: develop locally, commit, and push to master to deploy code into your Pantheon Development environment.
 
-## Resolving Conflicts
+## Resolve Conflicts
+
+<p>Conflicts can occur when the upstream you are trying to merge your code with has made alterations to files.</p>

```

Comment:
They were for spacing the following icon, if removed we'd just want to make sure it looks okay when rendered.